afrikaans/koeksisters/koeksisters.md

```diff
@@ -0,0 +1 @@
+As jy in Pretoria bly behoort jy te weet dat jy die beste koeksisters by [Springbok Koeksisters] (https://www.facebook.com/pages/Springbok-Koeksisters/799716243494753) kan koop.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.